bundles/mapping/mapwfs2/plugin/WfsVectorLayerPlugin.ol.js

```diff
@@ -20,7 +20,7 @@ export class WfsVectorLayerPlugin extends AbstractMapLayerPlugin {
         this._config = config;
         this.__name = 'WfsVectorLayerPlugin';
         this._clazz = 'Oskari.wfs.WfsVectorLayerPlugin';
-        this.renderMode = config.renderMode;
+        this.renderMode = config.renderMode || RENDER_MODE_VECTOR;
```

Comment:
The default value is set in init based on mapmodules 3D capabilities.
The change should be made there.

Reply:
Changed init to only change renderMode IF 3D mapmodule is used.